link to incidents on first reference (#4350)
camunda/camunda-docs: versioned_docs/version-8.5/components/modeler/bpmn/error-events/error-events.md

## Unhandled errors

> When an error is thrown and not caught, an [**incident**](/components/concepts/incidents.md) (i.e. `Unhandled error event`) is raised to indicate the failure. The incident is attached to the corresponding element where the error was thrown (i.e. the task of the processed job or the error end event).

Reviewer: ```suggestion
When an error is thrown and not caught, an [**incident**](/components/concepts/incidents.md) (for example, `Unhandled error event`) is raised to indicate the failure. The incident is attached to the corresponding element where the error was thrown (that is, the task of the processed job or the error end event).
```
Just a suggestion, spotted these i.e. instances

Author: Good little catch along the way 🧹
